certbot/plugins/disco.py

```diff
@@ -13,11 +13,11 @@
 from certbot import interfaces
 
 try:
-    from collections import OrderedDict
+    from collections import OrderedDict  # type: ignore
 except ImportError:  # pragma: no cover
     # OrderedDict was added in Python 2.7
-    from ordereddict import OrderedDict  # pylint: disable=import-error
-
+    from ordereddict import OrderedDict as _OrderedDict  # pylint: disable=import-error
+    OrderedDict = _OrderedDict  # type: ignore
```

Comment:
This is a *really* annoying way to work around https://github.com/python/mypy/issues/1393.  Ideally I wouldn't have to import the symbol as `_OrderedDict` on line 19, but I couldn't figure out how to comment that line with *both* a `# pylint:` *and* a `# type:` comment, so I had to do it in two lines. Bluggh.